packages/docregistry/src/context.ts

```diff
@@ -372,9 +372,11 @@ export class Context<T extends DocumentRegistry.IModel>
       let changeModel = change.newValue;
       // When folder name changed, `oldPath` is `foo`, `newPath` is `bar` and `this._path` is `foo/test`,
       // we should update `foo/test` to `bar/test` as well
+
       if (oldPath !== this._path) {
-        newPath = this._path.replace(new RegExp(`^${oldPath}`), newPath);
+        newPath = this._path.replace(new RegExp(`^${oldPath}/`), newPath + '/');
```

Comment:
For consistency, I would recommend using formatting strings:
```suggestion
        newPath = this._path.replace(new RegExp(`^${oldPath}/`), `${newPath}/`);
```

Reply:
no problem

Reply:
do I need a new pr or what can I do to direct change it without new pr?

Reply:
You should be able to click the "Commit suggestion" box here after first marking it back to unresolved.

Reply:
Thanks, I resolve it